### DISCUSSION TITLE
Connecting MindsDB to local MySQL Database

### DISCUSSION THREAD

**Discussion by flexicutioner** [mindsdb/mindsdb#9221] [Get help]
Hello all,

I am unable to connect a local MySQL server to my installation of MindsDB. I am getting the error "Can't connect to db: 2003 (HY000): Can't connect to MySQL server on '[127.0.0.1:3306](http://127.0.0.1:3306/)' (111)"

For reference, I have installed the MindsDB Lightwood distribution through Docker. I have also verified that the MySQL instance in question is indeed running on the machine. I have verified that the login information and database field are all correct. 

Thanks for the help!

**Comment by ZoranPandovski** [comment]
Hey @flexicutioner note that the localhost in the container is not the same as in your machine. Are you running MySQL with docker too?

**Reply by flexicutioner** [reply]
I am not running MySQL with docker, so that sounds like the issue. Thanks for the help!
